task02/task02/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task02
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите сторону правильного треугольника");
+            var storona = double.Parse(Console.ReadLine());
+
+            Console.WriteLine("Введите высоту пирамиды");
+            var visota = double.Parse(Console.ReadLine());
+
+            var ploschTreug = (Math.Pow(storona, 2) * Math.Sqrt(3)) / 4;
+            Console.WriteLine("Площадь равностороннего треугольника = " + Math.Round(ploschTreug, 2) + " квадратных единиц");
```

Comment:
Квадрат лучше вычислять умножением. Площадь основания пирамиды не требовалось выводить на консоль. Лишние скобки тоже ни к чему.
```suggestion
            var ploschTreug = storona * storona * Math.Sqrt(3) / 4;
```